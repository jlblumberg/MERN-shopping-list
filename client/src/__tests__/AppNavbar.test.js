import React from 'react';
import { shallow } from 'enzyme';
import AppNavbar from '../components/AppNavbar';

describe('AppNavbar', () => {
  const wrapper = shallow(<AppNavbar/>);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has a navbar with a github link', () => {
    expect(wrapper.find('#navbar').exists()).toEqual(true);
    expect(wrapper.find('#github-link').prop('href')).toEqual('https://github.com/jlblumberg/MERN-shopping-list');
  });

});