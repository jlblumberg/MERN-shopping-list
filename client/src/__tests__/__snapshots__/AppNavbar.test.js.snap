// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavbar renders correctly 1`] = `ShallowWrapper {}`;
